Code/Tools/AssetProcessor/native/resourcecompiler/RCBuilder.cpp

```diff
@@ -684,9 +684,19 @@ namespace AssetProcessor
                     ++retryCount;
                     ProcessLegacyRCJob(request, rcParam, assetRecognizer->m_productAssetType, jobCancelListener, response);
 
+                    // If a lost connection occured, prepare for a retry using an exponential backoff policy
+                    if ((response.m_resultCode == AssetBuilderSDK::ProcessJobResult_NetworkIssue) && (retryCount <= AssetProcessor::RetriesForJobLostConnection))
+                    {
+                        int delay = aznumeric_cast<int>(pow(2.0, aznumeric_cast<double>(retryCount-1)));
```

Comment:
I don't think you need that cast to `double` for the parameter to `pow`
```suggestion
                        const int delay = aznumeric_caster(pow(2, retryCount-1));
```